server/gokbg3/grails-app/domain/org/gokb/cred/TitleInstancePackagePlatform.groovy

```diff
@@ -13,7 +13,7 @@ import java.time.ZoneId
 class TitleInstancePackagePlatform extends KBComponent {
 
   static final String RD_PUBLICATION_TYPE = 'TitleInstancePackagePlatform.PublicationType'
-  static final String RD_MEDIUM = 'TitleInstancePackagePlatform.Medium'
+  static final String RD_MEDIUM = 'TitleInstance.Medium'
```

Comment:
Mixing of the two categories is not allowed.

Reply:
done